### PR TITLE
Rewriting in Python 3

### LINKED ISSUE #4: build option don't work
the option must be completely reviewed, as the path of previously generated markdown files is not saved anywhere.

### LINKED ISSUE #6: Add ability to customize the front page
Would be useful to add more text on the index.html page, to explain what people can find on the website

### LINKED ISSUE #7: Add possibility to generate documentation/reuse documentation from modules
Currently our modules are created in C++, thus ArkDoc can not read the source code and add it to the documentation.

It would be useful if ArkDoc could use the Markdown files from `lib/ext/module_name/documentation/` and include them in the generated html documentation.

### LINKED ISSUE #8: Generation is either incomplete or deleted
On the website the list documentation is incomplete, only 4 functions referenced while we have more than 15 in List.ark

### PR DESCRIPTION
**Pull request by SuperFola**
Closes
- #4 
- #6 
- #7 
- #8 
- ArkScript-lang/arkscript-lang.github.io#6
- ArkScript-lang/arkscript-lang.github.io#12

### CODE REVIEW

**Comment by SuperFola** [comment]
The single thing missing is the HTML generator from the list of functions (+ their documentation)

**Comment by SuperFola** [comment]
I implemented the C++ documentation thing, and added documentation merging: the idea is that, given String.ark (standard library) and String.cpp (builtins), the functions defined in both files will be in the same file group, to avoid having a String.cpp tab and a String.ark tab on the final website.

File: README.md [hunk 1]
```diff
@@ -1,74 +1,55 @@
...
+python3 -m venv venv
+source ./venv/bin/activate
+pip3 install -r requirements.txt
+python3 -m arkdoc --help
 ```
 
 ## Syntax
-You can be find the syntax ![here](./Syntax.md).
+
+- `@meta <name>`
```
Comment (SuperFola) [inline README.md:24]: Should we remove @meta since we now use the basename of the file (stripped of its extension)? @ArkScript-lang/lead-developers

**Comment by SuperFola** [comment]
Bug: no author
image

**Review by rakista112 — APPROVED** [review]
looks good

**Pull request merged**